cob_default_env_config/ipa-apartment/polygon_maps/restriction_polygon.yaml

```diff
@@ -0,0 +1,4 @@
+restricted areas: 
+  - id: 31287680
```

Comment:
there should be no restricted area in the apartment. I assume this is a left over from testing. Please remove.